data/json/items/generic/casing.json

```diff
@@ -246,3 +255,5 @@
     "damage_states": [ 0, 0 ]
   }
 ]
+
+
```

Comment:
What happened here?

Reply:
I kept getting trailing whitespace like that copying the web linter's output. Forgot to remove it in that file.